ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/presentation/ui/stepper/StepperScreen.kt

```diff
@@ -0,0 +1,33 @@
+package com.example.android.wearable.composeadvanced.presentation.ui.stepper
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import androidx.wear.compose.material.Stepper
+import androidx.wear.compose.material.Text
+
+/**
+ * Displays a Stepper
+ */
+@Composable
+fun StepperScreen(
+    displayValue: Int,
+    onValueChange: (Int) -> Unit
+) {
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .padding(top = 10.dp),
+        verticalArrangement = Arrangement.Center
+    ) {
+        Stepper(
+            value = displayValue,
+            onValueChange = onValueChange,
+            valueProgression = 1..10
+        ) { Text("Value: $displayValue") }
```

Comment:
A question for Compose team really, but should the button be disabled once max or min is hit?

Reply:
From an end user's perspective, i think a consistent user experience would be giving them some kind of indicator that they have reached the min/max when clicking the button. 

For slider, they can see visually that they have reached the min/max. However, for the stepper, as an user, I wouldn't know what's the min/max value. 

Disabling the stepper '+' or '-' button can implicitly let the users know they have reached the min/max. For consistency, it'd be good to disable the same buttons for slider, even though it has the visual indicator (the progress bar).